prime-angular-frontend/src/app/modules/enrolment/shared/services/enrolment-form-state.service.ts

```diff
@@ -332,6 +332,7 @@ export class EnrolmentFormStateService extends AbstractFormStateService<Enrolmen
 
   public buildOboSitesForm(): FormGroup {
     return this.fb.group({
+      lastUpdatedDatetime: [null, []],
```

Comment:
Please add a brief comment re: how this is used/why needed

Reply:
Comment added.